spec/factories/event_stream.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :event_stream do
+    sequence(:id) { SecureRandom.random_number(100) }
+    event_type      "TestEntry"
```

Comment:
If it's possible to make an `EventStream` without these attributes (I don't see any validations on them) it  may be better to omit them here - essentially leaving an empty factory